Add recipes for merging files & cropping video

kfrn opened the pull request:
Two new recipes:
* _Crop video_ - in the category **Change video properties**
* _Merge audio and video tracks_, in the **Other** category, next door to _Split audio and video tracks_.

These are pretty basic commands, so let me know if you think they're in scope ... and if so, whether I've 'filed' them correctly.

kfrn committed:
Add merge files command

kfrn committed:
Add Crop video recipe

retokromer commented on index.html:
```diff
@@ -666,6 +666,36 @@ <h2 id="video-properties">Change video properties</h2>
+        <dt>ffmpeg</dt><dd>starts the command</dd>
+        <dt>-i <i>input_file</i></dt><dd>path, name and extension of the input file</dd>
+        <dt>-vf "<i>width</i>:<i>height</i>"</dt><dd>Crops the video to the given width and height (in pixels).<br>
+          By default, the crop area is centred: that is, the position of the top left of the cropped area is set to x = (<i>input_width</i> - <i>output_width</i>) / 2, y = <i>input_height</i> - <i>output_height</i>) / 2.<br>
```
The `<br>` at the end is not needed.

retokromer commented:
Indeed, I think that they are in scope here!

kieranjol reviewed (commented):
Definitely within scope,these are great!I always end up googling every time I have to drop usually. My only suggestion relates to the mapping in the merge command. Should "stream" be pluralised to "streams" seeing as -map 0:v maps all videos streams from first input, -map 1:a maps all audio streams from second input?

ablwr commented:
Hell yeah!!!

kfrn committed:
Fix-ups: stray <br>, pluralisation

Also, I've unitalicised the argument to -map in the Split video command, since we are generally using italics for placeholders/variable names.

kfrn commented:
Nice catches @retokromer & @kieranjol ! I've made those changes. 🐶

retokromer reviewed (approved):
LGTM

retokromer commented:
Thank you very much again and again and again, @kfrn!

ablwr commented:
yay yaya yay

ablwr commented:
I am gonna hype ffmprovisr on amia-l
